.travis.yml

```diff
@@ -6,6 +6,11 @@ go_import_path: github.com/aws/amazon-ecs-agent
 
 install: make get-deps
 script:
+  - GOOS=linux GOARCH=amd64 go build -o ecs-agent ./agent
```

Comment:
Can you add this as a new Makefile target instead?  We pass a number of other flags to `go build` (linker flags, disabling cgo, etc) and it would be good to keep what we're testing in sync with the way we actually build our release binaries.